### DISCUSSION TITLE
Is anyone else always getting katas wrong in Codewars?

### DISCUSSION THREAD

**Discussion by Bangeloz** [codewars/codewars.com#3010] [Q&A]
Hi there,

I'm a Javascript learner, currently at a beginner level. A dev friend recommended that I gave a try to Codewars, so here I am trying a few exercises there (katas).

However, no matter what I try, when I check my answers Codewars always returns with error messages. I code in VSC, check everything is good, paste it in CW and then check the answer. That`s when the error messages pop up.

Does it happen to anyone else? Am I missing anything here? Cause I`ve been using CW for two days now and I`m starting to think that the problem is me LOL

Thanks in advance!

**Comment by hobovsky** [comment]
Please see if this explains anything: https://docs.codewars.com/training/training-example

**Comment by MinimumADHD** [comment]
Well, technically Codewars is intended to be a logical challenge-making platform, where instead of using a lot of libraries, you should use default content of the language your using. Did you try that?

**Comment by ewa-szataniak** [comment]
In JavaScript challenges like Codewars, it's best to use return statements instead of console.log for output. The goal is to write functions that solve specific problems and return the expected results. Test cases rely on the returned values to validate the correctness of your code. Using console.log may not produce the desired outcome and can interfere with the automated testing system. Focus on structuring your code to return the expected results from the function.

**Comment by raymondobroon** [comment]
Similar to @hobovsky comment, try reading through here too: [https://docs.codewars.com/training/troubleshooting](url)
